docs/guides/cloud/branch-review.mdx

```diff
@@ -83,15 +62,29 @@ If the selected branch has a PR associated and a PR # passed via CI, there is an
 
 ### View Options
 
-"Review" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration.
+"Overview" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration.
```

Comment:
* not just projects with GH integration – for all projects now
* I'd say default view "for a branch"